ingest/README.md

```diff
@@ -7,17 +7,18 @@ If you have another data source or private data that needs to be formatted for
 the phylogenetic workflow, then you can use a similar workflow to curate your
 own data.
 
-## Run
+## Workflow Usage
 
-From within the `ingest` directory, run the workflow with:
+All workflows are expected to the be run from the top level pathogen repo directory.
```

Comment:
Does this still work if we run `snakemake` from within `./ingest`, or does this break some paths? (I imagine paths such as `../shared/reference.fasta` are the most likely to not work.)

Reply:
I've just read pt. 1 in https://github.com/joverlee521/nextstrain-testing/blob/2650d10d1eb66a722469bcddda584bc1f50fae04/snakemake/shared-testing/current/README.md#different-nextstrain-build-commands -- it looks like the answer is that this won't work.

Reply:
> Does this still work if we run snakemake from within ./ingest, or does this break some paths? 

The paths restrictions only apply when using the containerized runtimes with `nextstrain build`, so using `snakemake` natively should still work. 

From within `./ingest`, you can run either `snakemake --cores 1` or `nextstrain build --conda`.

Reply:
That's great! Yes, good to be consistent and recommend the approach you want people to use, but it's a big win that people can jump into the ~siloed workflow directory and just run a normal snake-make pipeline. 

> The paths restrictions only apply when using the containerized runtimes with nextstrain build

Yup, but you can get around this using [this approach (internal slack link)](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1703709566142359?thread_ts=1703709079.972339&cid=C01LCTT7JNN) which is really nice.